substrate-node/pallets/pallet-dao/src/lib.rs

```diff
@@ -1,23 +1,16 @@
 #![cfg_attr(not(feature = "std"), no_std)]
 #![recursion_limit = "128"]
 
-use pallet_tfgrid::pallet::{InterfaceOf, LocationOf, SerialNumberOf, TfgridNode};
-use sp_runtime::traits::Hash;
+use pallet_tfgrid::pallet::{InterfaceOf, LocationOf, SerialNumberOf};
 use sp_std::prelude::*;
 
 use frame_support::{
     dispatch::{
-        DispatchError, DispatchResult, DispatchResultWithPostInfo, Dispatchable, GetDispatchInfo,
-        PostDispatchInfo,
+        DispatchResult, DispatchResultWithPostInfo, Dispatchable, GetDispatchInfo, PostDispatchInfo,
     },
-    ensure,
     traits::{EnsureOrigin, Get},
-    weights::Weight,
-};
-use tfchain_support::{
-    constants,
-    traits::{ChangeNode, Tfgrid},
 };
+use tfchain_support::traits::{ChangeNode, Tfgrid};
```

Comment:
Don't know if it is the best approach but I would try to get rid of all the `use` declarations in `lib.rs` and put them in the respective `mod` files for better partitioning
Keeping the strict necessary inside the pallet definition
~~~rust
#[frame_support::pallet]
pub mod pallet { ... }
~~~
what do you think @brandonpille ?

Reply:
Yes I think it's a good idea to minimize the imports.

Reply:
done in 90e68bbbcc48a06b91fdec8bf0bea6102a002251